src/features/operator/Market/MarketWithdraw/index.tsx

```diff
@@ -133,125 +133,59 @@ export const MarketWithdraw = (): JSX.Element => {
     }
   };
 
-  const baseToPreferredCurrency = React.useMemo(() => {
-    // Wait for prices to load (or error out) before displaying conversions
-    if (isLoadingPrices || isErrorPrices || prices === undefined) {
-      return '';
-    }
-
-    let amount = Big(0);
+  const convertAssetAmountToCurrency = (asset: Asset | undefined, amount: string) => {
```

Comment:
`(asset?: Asset, amount: string)`

Reply:
This helper function should better be in `./rates.api.ts` no?

Reply:
Agreed, yes.

The helper would need more arguments but at the end of the day is better to have a functional method.